src/Sylius/Bundle/AdminBundle/spec/Event/ManageCouponsMenuBuilderEventSpec.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\AdminBundle\Event;
+
+use Knp\Menu\FactoryInterface;
+use Knp\Menu\ItemInterface;
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\AdminBundle\Event\ManageCouponsMenuBuilderEvent;
+use Sylius\Component\Core\Model\PromotionInterface;
+
+final class ManageCouponsMenuBuilderEventSpec extends ObjectBehavior
+{
+    function let(FactoryInterface $factory, ItemInterface $menu, PromotionInterface $promotion): void
+    {
+        $this->beConstructedWith($factory, $menu, $promotion);
+    }
+
+    function it_is_a_manage_menu_builder_event(): void
+    {
+        $this->shouldHaveType(ManageCouponsMenuBuilderEvent::class);
```

Comment:
I would only specify that it should extend the `MenuBuilderEvent` class (`$this->shouldHaveType(MenuBuilderEvent::class);`) and get rid of checking a class name (the spec would not work anyway if it was wrong)